clients/apply/source_test.go

```diff
@@ -313,15 +313,15 @@ func TestTemplateSource_Read(t *testing.T) {
 	t.Parallel()
 
 	yamlTemplate := `---
-apiversion: influxdata.com/v2alpha1
+apiVersion: influxdata.com/v2alpha1
```

Comment:
I extracted this test data by using the `export` command to marshal YAML. Since the struct tags were missing, it marshalled using the wrong field name.